cmd/thanos/rule.go

```diff
@@ -130,6 +130,13 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 
 		lookupQueries := map[string]struct{}{}
 		for _, q := range *queries {
+			if q == "" {
+				return errors.New("Static querier address cannot be empty")
+			}
+			if strings.Contains(q, "://") {
```

Comment:
So we won't support `dns://` urls anymore?

Reply:
We never was suporting those. Only `dns+` URLS and others with `+`